pallets/randomness-beacon/README.md

```diff
@@ -0,0 +1,82 @@
+# Drand Bridge Pallet
+
+This is a [FRAME](https://docs.substrate.io/reference/frame-pallets/) pallet that allows Substrate-based chains to bridge to drand. It only supports bridging to drand's [Quicknet](https://drand.love/blog/quicknet-is-live-on-the-league-of-entropy-mainnet), which provides fresh randomness every 3 seconds. Adding this pallet to a runtime allows it to acquire verifiable on-chain randomness which can be used in runtime modules or ink! smart contracts. 
+
+Read [here](https://github.com/ideal-lab5/idn-sdk/blob/main/pallets/drand/docs/how_it_works.md) for a deep-dive into the pallet.
+
+## Usage
+
+Use this pallet in a Substrate runtime to acquire verifiable randomness from drand's quicknet.
+
+### Node Requirements
+
+Usage of this pallet requires that the node support:
+- offchain workers
+- (optional - in case of smart contracts) Contracts pallet and drand  chain extension enabled 
+
+We have included an example [solochain](https://github.com/ideal-lab5/idn-sdk/tree/main/pallets/drand/examples/solochain) in this repo, that meets these requirements that you can use to get started.
```

Comment:
@driemworks we probably wants to update the readme since it's mentioning the solochain

Reply:
I updated the readme, now it's really basic and just explains how to build + test the pallet.